packages/shared/src/write-archive/index.ts

```diff
@@ -36,7 +36,19 @@ export async function writeTestResult(
   archive: ResourceArchive,
   chromaticStorybookParams: ChromaticStorybookParameters
 ) {
-  const { title, outputDir, pageUrl } = e2eTestInfo;
+  const { titlePath, outputDir, pageUrl } = e2eTestInfo;
+  let titleHasTruncated = false;
+  const titlePathWithoutFileExtensions: string[] = [];
+  titlePath.forEach((aTitle) => {
+    // we only want to remove stuff after spaces for the file name
+    if (aTitle.includes('.') && !titleHasTruncated) {
+      titlePathWithoutFileExtensions.push(aTitle.split('.')[0]);
+      titleHasTruncated = true;
+    } else {
+      titlePathWithoutFileExtensions.push(aTitle);
+    }
+  });
+  const title = titlePathWithoutFileExtensions.join(' / ');
```

Comment:
CSF autotitle doesn't add spaces around ` / `, I'm not sure why you want to do that here.

```suggestion
  const title = titlePathWithoutFileExtensions.join('/');
```